lib/exercism/user_statistics.rb

```diff
@@ -0,0 +1,30 @@
+class UserStatistics
+  def self.of(username)
+    by_username(username)
+  rescue Exception => e
+    Bugsnag.notify(e, {username: username})
+    id
+  end
+
+  def self.by_username(username)
+    user = User.find_by(username: username)
+    { user: user.public_user_attributes, 
```

Comment:
Thanks @mhelmetag for the find. That was it! Looks like we are good to merge hopefully :)